packages/api/list-exports.ts

```diff
@@ -0,0 +1,5 @@
+import * as all from "./src";
```

Comment:
Please also delete this

Reply:
I removed this in c94ff3cacc315c168cf437a2f8d6b45a0f7c96b6